### ISSUE TITLE
SteamDeck installation screen resolution issue

### ISSUE THREAD

**Issue by Thundernerd** [ublue-os/bazzite#1004]
### Describe the bug

Tried to install Bazzite for SteamDeck, I believe version 36 but don't quote me on that, and the installation menu does not fit on the screen. It doesn't show all buttons/inputs and therefore makes it impossible to properly install without just winging it.

### What did you expect to happen?

I expected that the UI would fit on the screen
OR
That it would display on an external monitor (which I tried but unfortunately it doesn't)	

### Output of `rpm-ostree status`

```shell
N/A
```

### Hardware

SteamDeck 256GB LCD

**Comment by Dremor** [comment]
Same issue here, at least on Bazzite 3.0 Deck Gnome, buttons are way outside of the screen and no installation can be performed. Connecting a secondary screen did allow me to see a glimpse of the first button, and I was able to continue to the next step, but couldn't resize (at all) the Deck disk to fit Bazzite.

I'm currently downloading the KDE version to give it a try.

**Comment by ahoneybun** [comment]
I also see this on the Plasma version (I haven't tested the GNOME one) so +1 on my end.

**Comment by ctsdownloads** [comment]
Yeah, this appears to be the right place to file. Tagging @KyleGospo for visibility. Aaron is a friend of mine. :) Thanks Kyle!

**Comment by ahoneybun** [comment]
This is on version 39 based on the installer window.

**Comment by Stream5710** [comment]
Same issue happened to me with 3.0 Deck GNOME image on Steam Deck LCD 256GB.
By choosing Troubleshooting -> Basic Graphics Mode in grub menu it displayed on external display with correct resolution.

**Comment by Thundernerd** [comment]
Can confirm this as a workaround! Just tried it as well and was able to see the installation menu properly on my external monitor.

**Comment by TeamLinux01** [comment]
> Can confirm this as a workaround! Just tried it as well and was able to see the installation menu properly on my external monitor.

I ran into the same issue and workaround with 3.0.0 install image.

**Comment by NickFerg** [comment]
I’m having this same issue and don’t have an external monitor to connect to. Any suggestions on a workaround for installing directly on the deck?

**Comment by AkazaRenn** [comment]
The workaround doesn't work for me, I'm getting no output to the external display under basic display mode. Is there a place where we can get older versions of ISOs for a proper installer?

**Comment by Thundernerd** [comment]
@AkazaRenn 
IIRC someone mentioned using ventoy as a potential workaround

**Comment by AkazaRenn** [comment]
> @AkazaRenn 
> IIRC someone mentioned using ventoy as a potential workaround

Running the installer inside Ventoy works, thank you so much!

**Comment by ahoneybun** [comment]
@Thundernerd @AkazaRenn do you mean the workaround with an external display or does it boot on the Steam Deck's screen correctly when booting from Ventoy?

**Comment by AkazaRenn** [comment]
> @Thundernerd @AkazaRenn do you mean the workaround with an external display or does it boot on the Steam Deck's screen correctly when booting from Ventoy?

You don't need an external display, Steam Deck screen shows the installer in correct scaling when it's launched from Ventoy

**Comment by ahoneybun** [comment]
Ah got it, thanks! I think it would be good to keep the issue open since using a normal tool like GNOME Disks still has the issue.

**Comment by Hermitter** [comment]
I have the same issue on Bazzite 3.01 `Steam Deck 256GB+ (OLED in development)` 

Adding the grub parameter mentioned here fixed it for me: https://github.com/ublue-os/bazzite/issues/812#issuecomment-1963104198

**Comment by GuardianLiarus** [comment]
> I have the same issue on Bazzite 3.0.1 `Steam Deck 256GB+ (OLED in development)`
> 
> Adding the grub parameter mentioned here fixed it for me: [#812 (comment)](https://github.com/ublue-os/bazzite/issues/812#issuecomment-1963104198)

For those unfamiliar with that step, it means on the first option in grub, press "e" and then add that option after "quiet"

**Comment by Ferripro321** [comment]
I personally can't get Bazzite v3.6.0 to adjust to the screen size on installation with the steam deck oled, I tried everything...

When using ventoy I saw that the command `inst.resolution=1280x800` was already added to the parameters on grub but it doesn't work.

I tried using other flashers like Fedora Media Writer and the only difference was that the parameter on grub was not added and I had to manually enter it but even after adding it, it just doesn't work.

Setup always looks like the image on this issue: https://github.com/ublue-os/bazzite/issues/812#issue-2152886185

I also tried this with both the GNOME and KDA image and both seem to be having this issue, im not sure if this is a problem of the latest release or what but it's really strange as I tried everything on here and no results. (I didn't try using external screen as I don't have a dock)
